src/controllers/userController.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable valid-jsdoc */
+import Helper from '../helpers/helperUtils';
+import userService from '../services/userService';
+/**
+ * @class UsersController
+ */
+class UserController {
+  /** Login User
+   * @static
+   * @params {*} req
+   * @params {*} res
+   * @returns {object} loginUsers
+   */
+  static async loginAUser(req, res) {
+    const { email, password } = req.body;
+    try {
+      const loggedUser = await userService.loginAUser(email);
+      if (loggedUser) {
+        const validPassword = Helper.verifyPassword(loggedUser.password, password);
+        if (validPassword) {
+          const token = Helper.generateToken(loggedUser.dataValues);
```

Comment:
You are generating token with the entire user info, including createdAt and updatedAt which isnt necessary. The token should only contain specifics about the user that might be needed. E.g id, roleId.......

Reply:
ok I will fix that